CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # CHANGELOG
 
+## 1.25.2.dev0 (2022-05-13)
+
+MLflow 1.25.2.dev0 includes several major features and improvements:
+
+Features:
+
+- Add endpoint changes to the deployment CLI. (#5731, @trangevi)
+- Dev env setup script (#5717, @BenWilson2)
```

Comment:
Add development environment setup script

Reply:
Add development environment setup script for Linux and MacOS x86 Operating Systems